lib/handlers/passwordReset/SAResetV1.js

```diff
@@ -0,0 +1,23 @@
+const {getEmailAdapter} = require('./../../infrastructure/email');
+
+const process = async (config, logger, data) => {
+  const email = getEmailAdapter(config, logger);
+  await email.send(data.email, 'sa-reset-password', {
+    firstName: data.firstName,
+    lastName: data.lastName,
+    returnUrl: `${config.notifications.profileUrl}/register/${data.invitationId}`,
+  }, 'DfE Sign-in: Migrating your account');
```

Comment:
As the user has initiated this from a password reset then this subject may be confusing. It might make more sense to use the same subject as the main password reset?